snapcraft-assets/snap/command-chain/alsa-launch

```diff
@@ -7,44 +7,53 @@ else
 fi
 
 popup() {
-  TEXT="$SNAP_READABLE_NAME requires access to the ALSA system.
+  TEXT="$SNAP_READABLE_NAME may have improved audio quality if granted permission to access the ALSA audio system.
```

Comment:
I'm not sure audio quality is a problem that using alsa directly would necessarily resolve. It's more a case that you can't choose input and output devices when piping alsa through pulseaudio. (note that the application still uses alsa api but the alsa library is configured to forward that into pulseaudio)

```suggestion
  TEXT="$SNAP_READABLE_NAME may have better audio device support if granted permission to access the ALSA audio system.
```